Applications/SimpleShot/test/meson.build

```diff
@@ -15,5 +15,5 @@ exe = executable(
   install: get_option('enable-test'),
   install_dir: application_install_dir
 )
-test('simpleshot_tests', exe)
+test('simpleshot_tests', exe, args: '--gtest_output=xml:@0@/@1@.xml'.format(meson.build_root(), 'simpleshot_tests'))
```

Comment:
this is fine for now, but later you can consider locating all these `xml files` in a folder as the current `build root` is already very crowded.

Reply:
I'd like to, for now our ci reads xml from the build root so, ci needs to be changed altogether.

Reply:
then this can be done later.